src/systems/filecoin_blockchain/storage_power_consensus/storage_power_actor_state.go

```diff
@@ -10,6 +10,20 @@ import (
 	util "github.com/filecoin-project/specs/util"
 )
 
+func (st *StoragePowerActorState_I) PowerIsSmallerThanMin(minPower block.StoragePower) bool {
```

Comment:
Is `minPower` supposed to be miner power? A very confusing name in the context.

I think this functionality would be clearer inverted: `PowerMeetsConsensusMinimum()`

Reply:
We should also be more explicit here with regard to what Power we are talking about? TotalPower, ActivePower, or InactivePower? I think we should talk through this.